Helper/DbConnectionParamsNormalizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TheCadien/SuluImportExportBundle.
+ *
+ * (c) Oliver Kossin
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace TheCadien\Bundle\SuluImportExportBundle\Helper;
+
+class DbConnectionParamsNormalizer
+{
+    /**
+     * Normalizes the array with the connection params to a defined structure.
+     *
+     * @param array $connectionParams
+     *        The database connection params for the connection. Should be passed
+     *        as an associative array with the key 'url' or with the keys 'host',
+     *        'user', 'dbname' and 'password'.
+     *        If an URL is used, it should follow the format
+     *        'mysql://<user>:<password>@<host>/<dbname>'.
+     *
+     * @throws \InvalidArgumentException if the passed array does not contain the required fields
+     *
+     * @return array Returns an array with the keys 'host', 'user', 'dbname' and 'password'.
+     *         The latter may be null.
+     */
+    public static function normalize(array $connectionParams): array
```

Comment:
We should check `driver` from the connection params and throw an error whenever an incompatible driver is used. As far as I can tell, only `pdo_mysql` and `mysqli` should be supported right now.

Realistically, people are unlikely to make a mistake here, but when there are setups with multiple databases, we should abort early on.